src/NLog/Config/LoggingConfigurationFileLoader.cs

```diff
@@ -429,6 +432,26 @@ public IEnumerable<string> GetPrivateBinPathNLogLocations(string baseDirectory,
             }
         }
 
+#if NETSTANDARD && !NETSTANDARD1_3
+        private static bool SkipInvalidProcessDirectory(string processDirectory, string entryAssemblyLocation, IAppEnvironment appEnvironment)
```

Comment:
proposal name: "IsValidProcessDirectory". (if I understood the code correctly)

Reply:
Fixed